lib/commands/getAppConfig.js

```diff
@@ -0,0 +1,28 @@
+const wsContentTypes = require('../api/wsContentTypes');
+const chainPromise = require('../utils/chainPromise');
+const {getAppConfigRequest, getAppConfigResponse, getAppConfigError} = require('../texts');
+
+async function getAppConfig({appContext, authContext, webSockets, logger}) {
+	logger.log(getAppConfigRequest(appContext._context.configId));
```

Comment:
is this debug code?

Reply:
No. This code is display corresponding progress message to console, while code is running.